dashboard/test/factories.rb

```diff
@@ -475,6 +475,7 @@
   end
   factory :plc_course, class: 'Plc::Course' do
     name "MyString"
+    id Plc::Course.maximum(:id).try(:next) || 1
```

Comment:
Don't you want this to be a [lazy attribute](http://www.rubydoc.info/gems/factory_girl/file/GETTING_STARTED.md#Lazy_Attributes) evaluated each time? As it stands, it's static and should return the same value if it's called multiple times after initialization. I would expect this to overwrite id: 1 every time.